### DISCUSSION TITLE
Time machine; Shared folder; 10.6.8

### DISCUSSION THREAD

**Discussion by David72645** [Netatalk/netatalk#1602] [Q&A]
I'm trying to use Time Machine with a Mac running OS 10.6.8 (“Nikon”) to back up to a shared folder that is an external drive connected to another Mac (“Server”) running OS 14.7 on my home LAN.

Before I installed Netatalk, Nikon could see shared folders on Server but Nikon's Time Machine could not see the shared folder on Server.  A third Mac also running OS 14.7  (“iMac”) can see all the shared folders on Server, including in Time Machine at all times.

I installed Netatalk 4 on Server but Nikon still can't see the shared folder in Time Machine but now Nikon no longer sees the shared folders on Server. Instead Nikon sees a single folder called "users home".  iMac can still see the shared folders on Server and back up to the shared folder via Time Machine.

This is the text of  the afp.con file located at /usr/local/etc/afp.conf

```
[Global]
# Global configuration settings
log level = default:info
afp listen = 0.0.0.0

[Homes]
# This section is for user home directories
basedir regex = /Users
time machine = no

[Time Machine]
# Time Machine configuration for external drive
path = /Volumes/Nikon\ Time\ Machine
time machine = yes
```

After I stop Netatalk, nothing changes from when it was running. Time Machine still not working on Nikon and the shared folders that were visable before Netatalk are still replaced by “user’s home” folder.

**Comment by rdmark** [comment]
Is there a particular reason for the `afp listen = 0.0.0.0` line? I'm not sure if advertising the AFP service on this address is correct on macOS.

Regarding the issue you're having, when you say "can't see shared folders", I assume you mean that they're not picked up by Zeroconf automatically? When this happens, are you still able to access them by manually inputting the IP address of the server?

**Comment by APCCV** [comment]
I have never tried adding Netatalk to a Mac, let alone a modern one on Sonoma. I´d assume it supports it natively, but if you opened this thread maybe Sonoma doesn't support AFPS as a server anymore.

I'd try something like this:

```[Global]
# Global configuration settings
log level = default:info

[Homes]
# This section is for user home directories
basedir regex = /Users

[Time Machine]
# Time Machine configuration for external drive
path = /Volumes/Nikon\ Time\ Machine
vol dbpath = /Volumes/Nikon\ Time\ Machine\ DB
time machine = yes
valid users = @users 
```
I think the `valid users` tag is required and a matching group (users?) should exist or be created for it.
I also suggest you explicitly call for a location of the DB instead of letting Netatalk stashing wherever it might.
Last, path in /Volumes should have very open permissions (777 if you are OK with it, temporarily) until you get the connection going and then can tighten based on group.

With that `Homes` basedir, in best of theories, you should be able to connect to the home share (if it exists) and write files, which is a nice sanity test before trying out TimeMachine.

As for finding the shares in Finder, Avahi (or mDNSResponder) needs to know about this, which probably doesn't unless you compiled with ZeroConf support. I'd try to `Connect to Server` in Finder first, probably by IP.
